### PR TITLE
PUD-410:  Couple of additional pact tests for sentencing info

### PR DESCRIPTION
**Pull request by mikedunfordEE**

### CODE REVIEW

**Review by AndyNewton — APPROVED** [review]
Approved but conflicts need resolving.

File: fake-manage-recalls-api/stubs/__files/get-recall-with-sentencing-info.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "recallId": "8ab377a6-4587-2598-abc4-98fc53737",
+  "nomsNumber": "A1234AA",
+  "sentenceDate": "2019-08-03",
+  "licenceExpiryDate": "2021-08-03",
+  "sentenceExpiryDate": "2021-02-03",
+  "sentencingCourt": "Manchester Crown Court",
+  "indexOffence": "Burglary",
+  "conditionalReleaseDate": "2021-12-03",
+  "sentenceLength": {
```
Comment (jonnywyatt) [inline fake-manage-recalls-api/stubs/__files/get-recall-with-sentencing-info.json:10]: this is already in the get-recall stub?

Comment (AndyNewton) [reply fake-manage-recalls-api/stubs/__files/get-recall-with-sentencing-info.json:10]: (Aside!  ^^^ Perhaps an example for review in our "to be arranged" discussion of testing in this project? - per our Dev get-together of Wednesday)?

**Review by AndyNewton — APPROVED** [review]
LGTM: perhaps @jonnywyatt 's comment/question re. stub duplication can be used as an example in TBA team discussion?

File: fake-manage-recalls-api/stubs/__files/get-recall-with-sentencing-info.json (hunk 1 above)
Comment (jonnywyatt) [reply fake-manage-recalls-api/stubs/__files/get-recall-with-sentencing-info.json:10]: I'm confused :)
I'm just saying there's no need to create a new stub JSON file, we already have - https://github.com/ministryofjustice/manage-recalls-ui/blob/main/fake-manage-recalls-api/stubs/__files/get-recall.json#L30-L34

Comment (mikedunfordEE) [reply fake-manage-recalls-api/stubs/__files/get-recall-with-sentencing-info.json:10]: That depends on what exactly you want to test.  Our usage of that API at the moment would never return a fully populated Recall after saving sentencing info.  I discussed this briefly with Andy, and we need to agree our level of testing with Pact.

**Review by AndyNewton — APPROVED** [review]

File: fake-manage-recalls-api/stubs/__files/get-recall-with-sentencing-info.json (hunk 1 above)
Comment (jonnywyatt) [reply fake-manage-recalls-api/stubs/__files/get-recall-with-sentencing-info.json:10]: @AndyNewton you mentioned 'maximal' and 'minimal' examples, which sounds right to me. If that's the case, there's no need for an additional stub, as the existing get-recall.json stub is the 'maximal' example.

**Pull request merged**